components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -559,7 +562,10 @@ protected void initFile(String id) throws FormatException, IOException {
       int descrLength = in.readInt() * 2;
 
       if (blockLength > 0) {
-        offsets.add(in.getFilePointer() + descrLength);
+        String memBlockID = DataTools.stripString(in.readString(descrLength));
+        offsets.add(in.getFilePointer());
+        offsetBlockIDs.add(memBlockID);
+        descrLength = 0;
```

Comment:
Took me several iterations to read as either `descrLength` or `blockLength` will be `0` in the following `in.seek()` call.

Wonder if we should always call `String memBlockID = DataTools.stripString(in.readString(descrLength));` and have `in.seek(in.getFilePointer() + blockLength)`

Reply:
Fair point, and done in 8aeece0139.